src/_guides/language/sound-problems.md

```diff
@@ -240,8 +240,8 @@ For more information, see [Use proper input parameter types when overriding meth
 **Warning:**
 <code>Unsound implicit cast from <em>Class&lt;dynamic&gt;</em> to <em>Class&lt;type&gt;</em>.</code>
```

Comment:
Should I remove the entire "Unsound implicit downcast" section? https://kw-www-dartlang-1.firebaseapp.com/guides/language/sound-problems#unsound-implicit-downcast